data.braincommons.org/manifest.json

```diff
@@ -211,8 +211,8 @@
     },
     "presigned-url-fence": {
       "strategy": "auto",
-      "min": 2,
-      "max": 4,
+      "min": 20,
+      "max": 25,
```

Comment:
We should probably set up the regular fence to:
```
      "min": 5,
      "max": 15,
```
Just like we did in:
https://github.com/uc-cdis/cdis-manifest/blob/master/gen3.datastage.io/manifest.json#L346

What do you think?